src/librustc_target/abi/mod.rs

```diff
@@ -241,17 +241,18 @@ pub struct Size {
 }
 
 impl Size {
-    pub const ZERO: Size = Self::from_bytes(0);
+    pub const ZERO: Size = Size { raw: 0 };
 
     #[inline]
-    pub fn from_bits(bits: u64) -> Size {
+    pub fn from_bits(bits: impl TryInto<u64>) -> Size {
+        let bits = bits.try_into().ok().unwrap();
```

Comment:
Doesn't this result in a worse panic message than if you remove the `.ok()`?

Reply:
Yeah but it doesn't compile without the `ok()` as we don't know if the error type implements `Debug`.